<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainMenu</class>
 <widget class="QMainWindow" name="AdminMainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>冬奥纪念品交易平台-管理员主菜单</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="show_all_mers_btn">
       <property name="text">
        <string>查看所有商品</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="select_mer_btn">
       <property name="text">
        <string>搜索商品</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_mer_btn">
       <property name="text">
        <string>下架商品</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_all_orderforms_btn">
       <property name="text">
        <string>查看所有订单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_all_usrs_btn">
       <property name="text">
        <string>查看所有用户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ban_usr_btn">
       <property name="text">
        <string>封禁用户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_btn">
       <property name="text">
        <string>注销</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="select_label">
          <property name="text">
           <string>搜索栏：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="select_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="select_sure_btn">
          <property name="text">
           <string>进行搜索</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>🐕请输入商品名称进行检索</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
